deploy/future

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+# Don't run this with -e, grep exits with 1 when it finds no results
+
+function check_py () {
+# This is a bit janky. If you know how to get the output
+# to go on separate lines, please tell me.
+    results=`git --no-pager grep $1|grep "\.py"|grep -v "future-ok"`
+    if [ "$results" ]; then
+	echo $results
```

Comment:
Although, actually, instead of capturing the output, can you just do `git grep ... || EXIT=1` at the toplevel, and skip the reprinting and conditional entirely?  (This might also fix the `-e` issue.)